android/src/main/java/com/flutter_webview_plugin/WebviewManager.java

```diff
@@ -188,6 +190,21 @@ public void onReceiveValue(Boolean aBoolean) {
         }
     }
 
+    void takeScreenshot(MethodCall call, MethodChannel.Result result){
+        if (webView != null) {
+            webView.enableSlowWholeDocumentDraw();
+            webView.setDrawingCacheEnabled(true);
+            Bitmap bitmap = Bitmap.createBitmap(webView.getDrawingCache());
```

Comment:
It seems that getDrawingCache is deprecated (https://developer.android.com/reference/android/view/View.html#getDrawingCache())
Look at the deprecation comment at this link, they suggest two alternatives.

Reply:
`
For software-rendered snapshots of a small part of the View hierarchy or individual Views it is recommended to create a Canvas from either a Bitmap or Picture and call draw(android.graphics.Canvas) on the View. However these software-rendered usages are discouraged and have compatibility issues with hardware-only rendering features such as Config.HARDWARE bitmaps, real-time shadows, and outline clipping. `

I initially used this method but there were render bugs which caused 30% from top of the image to be a blank area.

I'll try the [PixelCopy API](https://developer.android.com/reference/android/view/PixelCopy.html).

Reply:
@roeierez Changed the deprecated method: https://github.com/breez/flutter_webview_plugin/pull/2/commits/98e8f8434b9fab07aec60db62ab4a62f9809a732